src/main/java/org/mule/extension/runtime/Expirable.java

```diff
@@ -0,0 +1,20 @@
+package org.mule.extension.runtime;
+
+/**
+ * An entity which can or cannot be considered elegible for expiration
```

Comment:
This part is not clear to me "An entity which can or cannot be considered". To me an 'expirable'  thing is something that "expires according to some expiration policy"